geonode_mapstore_client/client/js/api/geonode/adapter/index.js

```diff
@@ -22,14 +22,11 @@ import { parseDevHostname } from '@js/utils/APIUtils';
 * @return {promise} it returns an object with the success map object response
 */
 export const creatMapStoreMap = (body = {}) => {
```

Comment:
Move all the request to the api/geonode/v2 file changing the structure based on the methodology used by v2 requests (see createGeoStory, getGeoStoryByPk and updateGeoStory as an example)